packages/data/src/redux-store/index.js

```diff
@@ -631,7 +636,21 @@ function mapSelectorWithResolver(
 					metadataActions.failResolution( selectorName, args, error )
 				);
 			}
-		}, 0 );
+		} );
+
+		// Many resolvers can be called at once. The point of this is to at
+		// least batch `startResolution` actions all together.
```

Comment:
Why wouldn't you want the actions to be batched? Does it matter?

Reply:
Batching pauses all store listeners while the batch is processed. In fact this is the only thing that it does. And that changes what the store observers see. Instead of seeing some value go from `0` to `1` and then back to `0`, it will be flat `0`. Instead of seeing thing A change in one update and then thing B change in another update, you'll see A and B both change in one update.

All I'm saying is that user of the library will notice this, it will break someone's code and they will come here to complain 🙂

Reply:
Ok let's only batch our actions then 👍

Reply:
Ok, I separated these actions and changed it to only batch `startResolution` actions together.